### ISSUE TITLE
Subscript not shifted to left when base symbol is enclosed in brackets

### ISSUE THREAD

**Issue by jalonsomartinez** [KaTeX/KaTeX#3063]
**Describe the bug:**
The subscript is not shifted to the left to correct for italic, if the base symbol is enclosed in brackets.

**Expected behavior:**
I think `{V}_R` should render the same as `V_R`, as there is no semantic difference between the two. Maybe for groups of several symbols it makes sense, as in `{AV}_R`, although I'm not sure either.
This is especially important regarding computer-generated latex, where brackets are usually included even for single symbols.
This is the Mathjax behaviour.

**Screenshots:**
![image](https://user-images.githubusercontent.com/4263604/120310001-30a76a00-c2d6-11eb-8c4c-cf63c149caa4.png)
![image](https://user-images.githubusercontent.com/4263604/120310093-487eee00-c2d6-11eb-92d8-d5ecf6db3f01.png)

**Environment (please complete the following information):**
Tested in katex.org in various browsers

**Comment by jalonsomartinez** [comment]
Also relevant when using `\html*` commands, for example:
![image](https://user-images.githubusercontent.com/4263604/120328723-660a8280-c2eb-11eb-9622-02a9edf34751.png)

**Comment by edemaine** [comment]
I confirmed that LaTeX indeed doesn't visually distinguish between `V_R` and `{V}_R`.  @ronkok, do you understand the subscript mechanism to know why this happens?

**Comment by jalonsomartinez** [comment]
Don't know if this helps, but it seems to me that correcting for italic is done on purpose only for single symbols (and apparently some integrals):
https://github.com/KaTeX/KaTeX/blob/026bdf9f855c81467f262288061028c76d5d9db9/src/functions/supsub.js#L116-L129

**Comment by edemaine** [comment]
Yes, I also wondered whether this might be caused by #2987 (a recent change). I haven't tested this theory.
